src/Output.js

```diff
@@ -16,11 +16,14 @@ const Output = (props) => {
     const noOfResults = props.noOfResults;
     let shortenedArray = movieArray.slice(0, noOfResults ) 
     let newArray = shortenedArray; 
-    const sortCriteria = props.sortCriteria;
 
-    if (sortCriteria) {
+    if (props.sortCriteria) {
+
+      let sortArray = props.sortCriteria.split(' ')
```

Comment:
I'd consider organising the sort methods using an [enum](https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/) here, although unfortunately enums aren't natively supported in JavaScript (unlike most programming languages).

Enumerations are very useful when you want to specify a finite set of values, and also have the advantage that you can encapsulate information in one place. For example, you could specify the message displayed to a user and the sorting function within the same object:

```js
const sortMethods = {
  TITLE_ASCENDING : {
    userMessage: "Title A-Z",
    comparator: (a, b) => {
       return a > b;
    }
  },
  TITLE_DESCENDING : {
    userMessage: "Title Z-Z",
    comparator: (a, b) => {
       return b > a;
    }
  }
  // define other sort methods here
};
```

The default sort method of `TITLE_DESCENDING` could then be stored as a prop, and updated whenever the user switches the dropdown. This would mean the only thing you'd need to update when adding new sort methods in future is the `sortMethods` object.

Reply:
I have tried to implement the above in the latest commit, hope its as it should be!